packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.graph.ts

```diff
@@ -15,10 +15,6 @@ import { Team } from '@microsoft/microsoft-graph-types';
  * @memberof Graph
  */
 export async function getAllMyTeams(graph: IGraph): Promise<Team[]> {
-  const teams = await graph
-    .api('/me/joinedTeams')
-    .middlewareOptions(prepScopes('Team.ReadBasic.All'))
-    .select(['displayName', 'id', 'isArchived'])
-    .get();
+  const teams = await graph.api('/me/joinedTeams').select(['displayName', 'id', 'isArchived']).get();
```

Comment:
Is it okay if we drop `prepScopes` here altogether?

Reply:
yeah - this function is only used in the teams channel picker. I added a check that will ask the user for the appropriate scopes before this function gets called - it won't get called unless the user has consented to the scopes